app/code/Magento/Checkout/Model/PaymentInformationManagement.php

```diff
@@ -152,7 +152,11 @@ public function savePaymentInformationAndPlaceOrder(
             $orderId = $this->cartManagement->placeOrder($cartId);
         } catch (LocalizedException $e) {
             $this->logger->critical(
-                'Placing an order with quote_id ' . $cartId . ' is failed: ' . $e->getMessage()
+                'Placing an Order failed (reason: '.  $e->getMessage() .')',
```

Comment:
I think "reason" might be moved to "context" too. Also, we could add boolean "is_guest_checkout" to the context as well.

Reply:
@magento run all tests

Reply:
With `reason` I'll submit that as a separate PR, as this might bring discussion to the table, and I want this one to be merged asap

Reply:
I think keeping reason as a part of the message is a good idea.